Content.Server/Paper/Plane/PaperPlaneSystem.cs

```diff
@@ -0,0 +1,210 @@
+using Content.Server.DoAfter;
+using Content.Server.UserInterface;
+using Content.Shared.Movement.Components;
+using Content.Shared.Physics;
+using Content.Shared.Tag;
+using Content.Shared.Throwing;
+using Content.Shared.Verbs;
+using Robust.Shared.Physics.Dynamics;
+using Robust.Shared.Utility;
+using System.Threading;
+using static Content.Shared.Paper.SharedPaperComponent;
+
+namespace Content.Server.Paper.Plane
+{
+    public sealed class PaperPlaneSystem : EntitySystem
+    {
+        [Dependency] private DoAfterSystem _doAfterSystem = default!;
+        [Dependency] private readonly TagSystem _tagSystem = default!;
+
+        public override void Initialize()
```

Comment:
Organize the handlers in the same order as the events are initialized in this method, also move all other methods below the handlers, makes reading the code easier